app/config/config_test.yml

```diff
@@ -1,5 +1,6 @@
 imports:
     - { resource: config_dev.yml }
+    - { resource: security_test.yml }
```

Comment:
not saying it should change, but wouldn't configured known values be enough (say in `parameters.yml`)?